### ISSUE TITLE
[HTML] Dot in attribute names

### ISSUE THREAD

**Issue by woble** [sublimehq/Packages#575]
Some JS frameworks, i.e. Aurelia, use dots in attribute names, e.g. `<div title.bind="item.title" repeat.for="item for items"></div>`. However Sublime Text HTML syntax doesn't recognize those as attributes, which are valid by [HTML standards](https://www.w3.org/TR/html-markup/syntax.html#attribute-name).

Fix: add `\.` to the regex.

```yaml
        - include: entities

  tag-generic-attribute:
    - match: '\s+(([a-zA-Z0-9:\-_\.]+)\s*(=)\s*)'
      captures:
        1: meta.attribute-with-value.html
        2: entity.other.attribute-name.html
```

**Comment by keith-hall** [comment]
I thought the spec stated that custom attributes have to start with `data-`... https://html.spec.whatwg.org/multipage/dom.html#attr-data-*

**Comment by wbond** [comment]
This should be resolved by d8b2961e18071f203369f21216b356bba924cab4 and cec55c3007709fc4d6ac1739e5c15eee9482b69c
